### ISSUE TITLE
Expand filters when no JS

### ISSUE THREAD

**Issue by jduss4** [CDRH/orchid#222]
`.no-js .collapse { display: block; }`

I think we will need to set a class of `.no-js` on the <body> or <html> or something if the user does not have JS, then we can apply some styles to different site elements that need to be expanded
